### ISSUE TITLE
Bug in position provider

### ISSUE THREAD

**Issue by Mantas0726** [tananaev/traccar-client-android#296]
Hi,

there is a bug in PositionProvider:
```
DistanceCalculator.distance(location.getLatitude(), location.getLongitude(), lastLocation.getLatitude(), location.getLongitude()) >= distance
```
should be: 
```
DistanceCalculator.distance(location.getLatitude(), location.getLongitude(), lastLocation.getLatitude(), lastLocation.getLongitude()) >= distance
```

**Comment by tananaev** [comment]
Good catch. Can you send a pull request?

**Comment by garyvdm** [comment]
Fixed  by 6e3812320cc350d9f0b584630f29fc187a2335aa . Let's close this issue.
